PolyglotSafariExtension/ContentScript/content.css

```diff
@@ -3,13 +3,21 @@
     --background: rgba(200, 200, 200, 0.7);
     --text: #2b2b2b;
   }
+
+  .polyglot__confirm-button img {
+    filter: invert(11%) sepia(0%) saturate(61%) hue-rotate(286deg) brightness(98%) contrast(84%);
```

Comment:
This is generated by https://stackoverflow.com/a/50942954/6176994
I don't know a better solution, except:
- convert icon to SVG and use `currentColor`
- or add icon image twice with the necessary colors

Reply:
svg sounds better, but for now this solution is plausible enough.